ansible/vagrant/Vagrantfile.Solaris10

```diff
@@ -23,11 +23,11 @@ Vagrant.configure("2") do |config|
     adoptopenjdkSol10.vm.synced_folder ".", "/vagrant"
     adoptopenjdkSol10.vm.network :private_network, type: "dhcp"
     adoptopenjdkSol10.vm.provision "shell", inline: $script, privileged: false
-    adoptopenjdkSol10.vm.boot_timeout = 600
+    adoptopenjdkSol10.vm.boot_timeout = 900
   end
   config.vm.provider "virtualbox" do |v|
     v.gui = false
     v.memory = 5120
-    v.customize ["modifyvm", :id, "--cpuexecutioncap", "50"]
+    v.customize ["modifyvm", :id, "--cpuexecutioncap", "75"]
```

Comment:
Does this make a significant difference to things? I am nervous about increasing this to a fairly high value, especially given that we often run more than one in parallel during VPC.

Reply:
It does reduce the amount of random failures/timeouts during the ansible play, let me try reducing it to 60 and testing that, it may be that now I've cleared down a lot of the leftover processes a slightly smaller increase might be ok. As its only the solaris VMs that have slightly more headroom, there shouldnt be too much of an impact... let me test it at 60....

Reply:
have reduced it to 60, its slower, but still stable.